test/Microsoft.Extensions.SecretManager.Tests/ConfigurationExtensionTests.cs

```diff
@@ -30,7 +31,7 @@ public void AddUserSecrets_Does_Not_Fail_On_Non_Existing_File()
         {
             var projectPath = UserSecretHelper.GetTempSecretProject();
 
-            var builder = new ConfigurationBuilder().SetProjectPath(projectPath).AddUserSecrets();
+            var builder = new ConfigurationBuilder().SetFileProvider(new PhysicalFileProvider(projectPath)).AddUserSecrets();
```

Comment:
You can use `SetBasePath(projectPath)` here. I will setup the file provider for you.